pandas/core/ops/__init__.py

```diff
@@ -433,7 +433,8 @@ def f(self, other, axis=default_axis, level=None, fill_value=None):
 
         if isinstance(other, ABCDataFrame):
             # Another DataFrame
-            new_data = self._combine_frame(other, na_op, fill_value)
+            with np.errstate(all="ignore"):
+                new_data = self._combine_frame(other, na_op, fill_value)
```

Comment:
shouldnt we get this one for free bc it goes through _dispatch_frame_op which you've got setting np.errstaet?

Reply:
Ah, yes, indeed `_combine_frame` eventually always goes through `_dispatch_frame` as well (I got here because it's a place that uses `get_array_op`)